public/css/1080p.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+    font-family: 'spiegel-bold';
+    src: url('../fonts/Spiegel-OTF/spiegel-bold.otf');
+}
+
+.main-window {
+    background-color: transparent;
+    color: #C89B3C;
+    font-family: 'spiegel-bold', serif;
+    font-size: 16px;
+}
+
+#total-gold-row {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 26.5vh;
+    color: #C89B3C;
+    row-gap: 50px;
+}
+
+#allied-gold {
+    margin-right: 2vw;
+}
+
+#enemy-gold {
+    margin-left: 2vw;
+}
+
+#top-lane {
```

Comment:
different naming convention. we have top-lane then rest is jungle, mid, adc, support